{{cookiecutter.plugin_name}}/README.md

```diff
@@ -1,10 +1,10 @@
-# napari-{{cookiecutter.plugin_name}}
+# {{cookiecutter.plugin_name}}
 
-[![License](https://img.shields.io/pypi/l/napari-{{cookiecutter.plugin_name}}.svg?color=green)](https://github.com/napari/napari-{{cookiecutter.plugin_name}}/raw/master/LICENSE)
-[![PyPI](https://img.shields.io/pypi/v/napari-{{cookiecutter.plugin_name}}.svg?color=green)](https://pypi.org/project/napari-{{cookiecutter.plugin_name}})
-[![Python Version](https://img.shields.io/pypi/pyversions/napari-{{cookiecutter.plugin_name}}.svg?color=green)](https://python.org)
-[![tests](https://github.com/{{cookiecutter.github_username}}/napari-{{cookiecutter.plugin_name}}/workflows/tests/badge.svg)](https://github.com/{{cookiecutter.github_username}}/napari-{{cookiecutter.plugin_name}}/actions)
-[![codecov](https://codecov.io/gh/{{cookiecutter.github_username}}/napari-{{cookiecutter.plugin_name}}/branch/master/graph/badge.svg)](https://codecov.io/gh/{{cookiecutter.github_username}}/napari-{{cookiecutter.plugin_name}})
+[![License](https://img.shields.io/pypi/l/{{cookiecutter.plugin_name}}.svg?color=green)](https://github.com/napari/{{cookiecutter.plugin_name}}/raw/master/LICENSE)
```

Comment:
just fyi one minor issue: this is hardcoded to napari org, and we probably want to use either github username or ask for the organization. If you want I can create a ticket for this

Reply:
can you submit a PR for it please?  change `https://github.com/napari/` to `https://github.com/napari/{{cookiecutter.github_username}}`